providers/go-feature-flag/src/main/java/dev/openfeature/contrib/providers/gofeatureflag/controller/CacheController.java

```diff
@@ -23,12 +24,11 @@ public class CacheController {
 
     @Builder
     public CacheController(GoFeatureFlagProviderOptions options) {
-        this.cache = options.getCacheBuilder() != null ? options.getCacheBuilder().build() : buildDefaultCache();
+        this.cache = options.getCacheConfig() != null ? options.getCacheConfig().build() : buildDefaultCache();
     }
 
     private Cache<String, ProviderEvaluation<?>> buildDefaultCache() {
-        return CacheBuilder.newBuilder()
-                .concurrencyLevel(DEFAULT_CACHE_CONCURRENCY_LEVEL)
```

Comment:
**suggestion:** Since we are not using the constant `DEFAULT_CACHE_CONCURRENCY_LEVEL` it would be great to delete it from the `CacheController`.

Reply:
Done!